src/server/index.js

```diff
@@ -4,47 +4,84 @@ const { createClient } = require('./sproutVideo');
 const { ELEMENT_STATE } = require('../shared');
 const omit = require('lodash/omit');
 
+async function beforeSave(asset, { config: { tce } }) {
+  const { sproutVideoApiKey: apiKey } = tce;
+  const client = createClient({ apiKey });
+  const { id: videoId, playable } = asset.data.video;
+  const isVideoPlayable = videoId && playable;
+  if (!isVideoPlayable) return asset;
+  await processCaption(asset, client);
+  return asset;
+}
+
+function processCaption(asset, client) {
+  const {
+    video: { id: videoId },
+    caption: { id: captionId, content, status }
+  } = asset.data;
+  delete asset.data.caption.content;
```

Comment:
Why do we need to delete this?

Reply:
The `caption.content` property is deleted before saving to the database. If the property is kept, it will be stored in the database but I don't think it's necessary.